crates/ide-assists/src/handlers/introduce_named_generic.rs

```diff
@@ -61,6 +67,9 @@ pub(crate) fn introduce_named_generic(acc: &mut Assists, ctx: &AssistContext<'_>
                     edit.add_tabstop_before(cap, generic_param);
```

Comment:
The weird snippet cursor position(`$0`) thing is caused in here.
As I commented on the first panic comment, our code is changed when the `SyntaxEditor::finish` is called.
So here, the last element of generic params is pre-existing one, not our new one. (Sorry for my misleading comment about this weeks ago 😅)
I think that we should change `add_tabstop_before` as entried like `add_file_edits` and triggered in order

Reply:
Oh, I was wrong again 😅 Sorry. I think that we should use `SyntaxEditor::add_annotation` here (Find its usages for examples)